sycl/doc/GetStartedWithSYCLCompiler.md

```diff
@@ -58,12 +151,28 @@ should be used:
 # Test the SYCL compiler and runtime
 
 Run LIT testing using the command below after building SYCL compiler and runtime.
-
+Use one of these commands:
+### Linux:
 ```bash
 make -j`nproc` check-all
+make -j`nproc` check-llvm
+make -j`nproc` check-llvm-spirv
+make -j`nproc` check-clang
+make -j`nproc` check-sycl
```

Comment:
All these new commands are executed as part of `check-all`, so I don't see much value in running both.

Reply:
check-all includes all others below, right?
The reason why I added those smaller checks is just to show they exist. New users may not know 'check-sycl' or 'check-llvm-spirv' exists.

It makes sense to clarify that moment somehow in this document.